examples/group_level/plot_continuous_covariate.py

```diff
@@ -0,0 +1,361 @@
+# compute low and high percentiles for bootstrapped beta coefficients
+lower_b, upper_b = np.quantile(boot_betas, [a/boot, c/boot], axis=0)
+
+# reshape to channels * time-points space
+lower_b = lower_b.reshape((n_channels, n_times))
+upper_b = upper_b.reshape((n_channels, n_times))
+
+###############################################################################
+# fit linear model with sklearn's LinearRegression
+# we already have an intercept column in the design matrix,
+# thus we'll call LinearRegression with fit_intercept=False
+
+# set up and fit model
+linear_model = LinearRegression(fit_intercept=False)
+linear_model.fit(group_design, betas)
+
+# extract group-level beta coefficients
+group_coefs = get_coef(linear_model, 'coef_')
+
+# only keep relevant predictor
+group_betas = group_coefs[:, group_pred_col]
+
+# back projection to channels x time points
+group_betas = group_betas.reshape((n_channels, n_times))
+# create evoked object containing the back projected coefficients
+group_betas_evoked = EvokedArray(group_betas, epochs_info, tmin)
+
+###############################################################################
+# plot the modulating effect of age on the phase coherence predictor (i.e.,
+# how the effect of phase coherence varies as a function of subject age)
+# using whole electrode montage and whole scalp by taking the
+# same physical electrodes across subjects
+
+# index of B8 in array
+electrode = 'B8'
+pick = group_betas_evoked.ch_names.index(electrode)
+
+# create figure
+fig, ax = plt.subplots(figsize=(7, 4))
+ax = plot_compare_evokeds(group_betas_evoked,
+                          ylim=dict(eeg=[-3, 3]),
+                          picks=pick,
+                          show_sensors='upper right',
+                          axes=ax)
+ax[0].axes[0].fill_between(times,
+                           # transform values to microvolt
+                           upper_b[pick] * 1e6,
+                           lower_b[pick] * 1e6,
+                           alpha=0.2)
+plt.plot()
+
+###############################################################################
+# run analysis on optimized electrode (i.e., electrode showing best fit for
+# phase-coherence predictor).
+
+# reshape subjects' R-squared to channels * time-points space
+r_squared = r_squared.reshape((r_squared.shape[0], n_channels, n_times))
+
+# look for electrode with the highest R-squared variance in each subject
+optimized_electrode = []
+for i in range(r_squared.shape[0]):
+    subj_r2 = r_squared[i, :, :]
+
+    # use electrode R-squared variance
+    # electrode_variance = []
+    # for e in range(subj_r2.shape[0]):
+    #     electrode_variance.append(np.var(subj_r2[e, :]))
+
+    # optimized_electrode.append(np.argmax(electrode_variance))
+
+    # or use max R-squared ?
+    optimized_electrode.append(np.unravel_index(r_squared[i, :, :].argmax(),
+                                                r_squared[i, :, :].shape)[0])
+
```

Comment:
I'm a bit confused bz what you're doing here. Couldn't you do `optimized_electrodes = r_squared.argmax(axis=1)` and be done withit instead of the full loop?

Reply:
I'm looking for the electrode which achieved the R-squared peak in the whole temporal spectrum used or analysis. Doesn't mather if the peak was at .1 or .12 s. I just want the index of the electrode for each person in the data set, regarless of when the `argmax()` was acheived.
Hence:
1. I'm looping through the first dimesion of R-squared (the subjcets)
2. get the `argmax()`of the reamining two dimensions (electrodes * times).
3. transform the result (wich is flat) to ist original shape (electrodes * times) and get the electrode (i.e., `[0]`).

using `optimized_electrodes = r_squared.argmax(axis=1)` as you suggested, would give me the max electrode at each time of the analyisis time window. Then we would still need find the oveall max.

Reply:
but I could use something like you suggest:
`optimized_electrodes = r_squared.argmax(axis=1)`

and then unravel to the original shape to get the electrode:
`np.unravel_index(optimized_electrodes, (n_channels, n_times))[0]`

would still simplify the code considerably, I guess.